include/librealsense2/dds/dds-device-broadcaster.cpp

```diff
@@ -241,16 +237,16 @@ bool dds_device_broadcaster::create_device_writer( const std::string &device_key
     return _device_handle_by_sn[device_key].data_writer != nullptr;
 }
 
-bool dds_device_broadcaster::create_dds_publisher()
+void dds_device_broadcaster::create_dds_publisher()
```

Comment:
I think I like `create_broadcast_topic` as a better name... what do you think? A `broadcaster` is something more... implementation.

Reply:
👍